main.py

```diff
@@ -1,33 +1,30 @@
-from flask import Flask, render_template
+from flask import Flask, jsonify, make_response, render_template, request
 from dotenv import dotenv_values
-from random import choice
+from db import connectDb
 import urllib.parse
 import requests
 import json
-import os
 
 app = Flask(__name__)
 
 config = dotenv_values('.env')
 OPENCAGEDATA_API_KEY = config.get('OPENCAGEDATA_API_KEY')
 GEOAPIFY_API_KEY = config.get('GEOAPIFY_API_KEY')
+SECRET_KEY = config.get('SECRET_KEY')
 
-POPULATION_THRESHHOLD = 400000
+POPULATION_THRESHHOLD = 1000000
 
-class City:
-    def __init__(self, name, coords):
+class CityInfo:
+    def __init__(self, name, coords, country, cityId):
         self.name = name 
         self.coords = coords 
+        self.country = country
+        self.cityId = cityId
 
 def getRandomCityName(populationThreshold):
-    with open('city_data/worldCities.json', 'r', encoding='UTF-8') as r:
-        cities = json.loads(r.read())
-        nonEmptyPopulation = list(filter(lambda city: (city['population'] != ""), cities))
-        filteredCities = list(filter(lambda city: (city['population'] > populationThreshold), nonEmptyPopulation))
-        randCity = choice(filteredCities)
-        
-        city = str(randCity['city']).replace('"', '')
-        return city 
+    city = connectDb.getCitiesByPopulation(populationThreshold)
+
+    return city
```

Comment:
Function now returns an entire CityInfo object, not a city name.

`getRandomCity` or `getRandomCityByPopulation`. Prolly the former /shrug